From 9c0f87aabffc195b58328c670b1bc19ab6ebe76d Mon Sep 17 00:00:00 2001
From: John Cox <jc@kynesim.co.uk>
Date: Fri, 3 Jul 2020 16:09:50 +0100
Subject: [PATCH 48/54] Add P030 defn in case it doesn't exist

---
 libavcodec/v4l2_request.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/libavcodec/v4l2_request.c b/libavcodec/v4l2_request.c
index 60f51e5ec4..4ca42d29ec 100644
--- a/libavcodec/v4l2_request.c
+++ b/libavcodec/v4l2_request.c
@@ -31,6 +31,12 @@
 #include "v4l2_request.h"
 #include "v4l2_phase.h"
 
+// P030 should be defined in drm_fourcc.h and hopefully will be sometime
+// in the future but until then...
+#ifndef DRM_FORMAT_P030
+#define DRM_FORMAT_P030 fourcc_code('P', '0', '3', '0')
+#endif
+
 uint64_t ff_v4l2_request_get_capture_timestamp(AVFrame *frame)
 {
     V4L2RequestDescriptor *req = (V4L2RequestDescriptor*)frame->data[0];
-- 
2.29.2

