From 54b4f2e72ab3bd84d4a185678e23706aa192122d Mon Sep 17 00:00:00 2001
From: John Cox <jc@kynesim.co.uk>
Date: Tue, 7 Jul 2020 15:05:34 +0100
Subject: [PATCH 51/55] Add unistd.h to define close

---
 libavcodec/rpivid_hevc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libavcodec/rpivid_hevc.c b/libavcodec/rpivid_hevc.c
index ffcca130e0..6a7e085da9 100644
--- a/libavcodec/rpivid_hevc.c
+++ b/libavcodec/rpivid_hevc.c
@@ -6,6 +6,7 @@
 #include <fcntl.h>
 #include <pthread.h>
 #include <semaphore.h>
+#include <unistd.h>
 #include <sys/mman.h>
 
 #include "fftools/ffmpeg.h"
-- 
2.29.2

